.vimrc

```diff
@@ -436,3 +452,33 @@ let g:matchup_matchparen_deferred = 1
 " Diminactive
 " ============================================================================
 let g:diminactive_enable_focus = 1
+
+" ============================================================================
+" ANYFOLDS
+" ============================================================================
+autocmd Filetype * AnyFoldActivate   
+set foldlevel=99 " Open all folds
+
+" ============================================================================
+" LENGTHMATTERS
+" ============================================================================
+let g:lengthmatters_start_at_column = 88
+
+" ============================================================================
+" FZF 
+" ============================================================================
+fun! s:fzf_root()
+	let path = finddir(".git", expand("%:p:h").";")
+	return fnamemodify(substitute(path, ".git", "", ""), ":p:h")
+endfun
+
+"nnoremap <silent> <space>f :exe 'Files ' . <SID>fzf_root()<CR>
+nnoremap <silent> <space>f :call fzf#run(fzf#vim#with_preview(fzf#wrap({'source': 'ag --hidden --ignore node_modules --ignore .git -g ""', 'dir': <SID>fzf_root()})))<CR>
+nnoremap <silent> <space><space> :call fzf#run(fzf#vim#with_preview(fzf#wrap({'source': 'ag --hidden --ignore node_modules --ignore .git -g ""'})))<CR>
+nnoremap <silent> <space>c :CocFzfList<CR>
+
+" ============================================================================
+" COC PYTHON 
+" ============================================================================
+
+call coc#config('python', {'pythonPath': split(execute('!which python'), '\n')[-1]})
```

Comment:
What does this do?

Reply:
there is a bug with coc-python it does not pic the correct interpreter when you are working with multiple envs, with this config we ensure that we are suing the correct python interpreter by env